controllers:
  ros__parameters:
    battery:
      topic: battery_state
      rate: 10.0
      error_battery_level: 11.5
      warn_battery_level: 12.2
      capacity_ah: 10.0
      battery_type: LiFePo4
      nominal_voltage: 13.2
